src/pybind/mgr/dashboard/frontend/src/app/shared/validators/cd-validators.ts

```diff
@@ -62,4 +62,18 @@ export class CdValidators {
       return success ? {'required': true} : null;
     };
   }
+
+  /**
+   * Checks if the value of password matches confirmPassword.
```

Comment:
Please add some information that the form controls must be named ``password`` and ``confirmPassword`` or enhance the validator to allow the developer to configure the control identifiers/paths.